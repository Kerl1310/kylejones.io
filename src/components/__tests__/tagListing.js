import React from 'react';
import renderer from 'react-test-renderer';

import TagListing from '../tagListing';

const testTag = {
    fieldValue: "aws",
    totalCount: 5
}

describe('Tag Listing', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<TagListing tag={testTag}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
