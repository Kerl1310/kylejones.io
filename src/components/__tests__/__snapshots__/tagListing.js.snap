// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Listing renders correctly 1`] = `
<li
  className="tag"
  id="aws"
>
  <a
    href="/tags/aws/"
  >
    aws
     (
    5
    )
  </a>
</li>
`;
